src/cascadia/WindowsTerminal/IslandWindow.h

```diff
@@ -89,4 +91,15 @@ class IslandWindow :
     std::function<void(const HWND, const RECT, winrt::TerminalApp::LaunchMode& launchMode)> _pfnCreateCallback;
 
     void _HandleCreateWindow(const WPARAM wParam, const LPARAM lParam) noexcept;
+
+    bool _fullscreen{ false };
+    RECT _rcFullscreenWindowSize;

```

Comment:
Aren't we avoiding Hungarian prefixes in our new code? `_rc` looks pretty Hungarian notation style to me.

Reply:
(and below with the `f`s)